project-2/Inconsisent_Data.ipynb

```diff
@@ -0,0 +1,5135 @@
+{
```

Comment:
<p>Add intermitent descriptions of what you're doing (comments and markdown descriptions)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/Humphery_Ufuoma/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>